### PR TITLE
Golint changes

### PR DESCRIPTION
**Pull request by Soundharyaam**
Fixes: https://github.com/openebs/openebs/issues/775

Fixed linting issues in maya/types/v1/meta.go

### CODE REVIEW

**Comment by ggarg2906sumo** [comment]
PR still contain .Ds binary files

**Comment by prateekpandey14** [comment]
@Soundharyaam please do the following 
1. Remove all .Ds files and 
2. Squash the commits to 1 commit

**Comment by Soundharyaam** [comment]
@kmova I have removed all the .DS_Store files.  But, still not passing through.

File: types/v1/meta.go [hunk 1]
```diff
@@ -256,11 +256,11 @@ type LabelSelectorRequirement struct {
 	Values []string `json:"values,omitempty" protobuf:"bytes,3,rep,name=values"`
 }
 
-// A label selector operator is the set of operators that can be used in a selector requirement.
+// LabelSelectorOperator is the set of operators that can be used in a selector requirement.
 type LabelSelectorOperator string
 
 const (
-	LabelSelectorOpIn           LabelSelectorOperator = "In"
+	//LabelSelectorOpIn           LabelSelectorOperator = "In"
```
Comment (kmova) [inline types/v1/meta.go]: @Soundharyaam .. why is this commented?

**Pull request merged**